Rakefile

```diff
@@ -0,0 +1,10 @@
+# encoding: utf-8
+
+require 'rake/testtask'
+
+Rake::TestTask.new do |task|
+  task.libs << %w(test lib)
+  task.pattern = 'tests/test_*.rb'
+end
+
+task :default => :test
```

Comment:
Add a rake task for testing, tell it where to find all the test files, and then set `test` as the default if no other rake argument is given.